### PR TITLE
feat(aggiemap-angular): Adds beta branch modal popup

### PR DESCRIPTION
**Pull request by Proantagonist**
Adds a one-time notification prompting the existence of an aggiemap beta branch. This is meant for production to guide users into dev.aggiemap.tamu.edu. 

image

On the dev branch, users are now greeted with a modal popup describing the differences between prod and dev.

image

### CODE REVIEW

File: apps/aggiemap-angular/src/environments/notification-events.ts [hunk 1]
```diff
@@ -59,5 +59,14 @@ export const NotificationEvents: NotificationProperties[] = [
     message: 'Thank you! Your feedback has been submitted successfully and the Aggiemap team notified.',
     imgUrl: './assets/images/like.svg',
     imgAltText: 'Feedback Icon'
+  },
+  {
```
Comment (atharmon) [inline apps/aggiemap-angular/src/environments/notification-events.ts:63]: At the beginning of `notification-events.ts` you have a commented out object, you saving that for a rainy day?

File: apps/aggiemap-angular/src/app/app.component.ts [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
+export class AppComponent implements OnInit {
+  constructor(
+    public analytics: Angulartics2GoogleAnalytics,
+    private readonly vcr: ViewContainerRef,
+    private readonly ms: ModalService
+  ) {
     analytics.startTracking();
   }
+
+  public ngOnInit(): void {
```
[the remark this answers is not in the document]
Comment (Proantagonist) [reply apps/aggiemap-angular/src/app/app.component.ts:20]: Doesn't matter. It was generated by a snippet.

File: apps/aggiemap-angular/src/environments/notification-events.ts (hunk 1 above)
Comment (Proantagonist) [reply apps/aggiemap-angular/src/environments/notification-events.ts:62]: Commented it in and removed the active date range and will only show when manually invoked.

**Review by atharmon — APPROVED** [review]
👍

**Pull request merged**